File: srp.js
class Journal {
    constructor() {
        this.entries = {};
    }

    addEntry(text) {
        let c = ++Journal.count;
        let entry = `${c}: ${text}`l
        this.entries[c] = entry;
        return c;
    }

    removeEntry(index) {
        delete this.entries[index];
    }

    toString() {
        return Object.values(this.entries).join(`\n`)
    }

}
module.exports = Journal;

File: srp.py
class Journal:
    def __init__(self):
        self.entries = []
        self.count = 0

    def add_entry(self, text):
        self.count += 1
        self.entries.append(f'{self.count}: {text}')

    def remove_entry(self, pos):
        del self.entries[pos]
        
    def __str__(self):
        return '\n'.join(self.entries)
    
    # def save(self, filename):
    #     file = open(filename, 'w')
    #     file.write(str(self))
    #     file.close()
        
    # def load(self, filename):
    #     pass
    
    # def load_from_web(self,uri):
    #     pass
    
class PersistenceManager:
    @staticmethod
    def save_to_file(journal,filename):
        file = open(filename, 'w')
        file.write(str(journal))
        file.close()
        
    def load(self, filename):
        pass
    
    def load_from_web(self,uri):
        pass